nextcloud-aio-helm-chart/update-helm.sh

```diff
@@ -218,6 +218,10 @@ find ./ \( -not -name '*service.yaml' -name '*.yaml' \) -exec sed -i "/^status:/
 # shellcheck disable=SC1083
 find ./ \( -not -name '*persistentvolumeclaim.yaml' -name '*.yaml' \) -exec sed -i "/resources:/d" \{} \; 
 # shellcheck disable=SC1083
+find ./ -name "*namespace.yaml" -exec sed -i "1i\\{{- if ne .Values.NAMESPACE default }}" \{} \; 
```

Comment:
```suggestion
find ./ -name "*namespace.yaml" -exec sed -i "1i\\{{- if ne .Values.NAMESPACE \"default\" }}" \{} \; 
```

The namespace name needs to be quoted. Otherwise, an error is thrown:
`Error: template: nextcloud-aio-helm-chart/templates/nextcloud-aio-namespace-namespace.yaml:1:28: executing "nextcloud-aio-helm-chart/templates/nextcloud-aio-namespace-namespace.yaml" at <default>: wrong number of args for default: want at least 1 got 0`